src/components/signUp/SignUpForm.tsx

```diff
@@ -78,66 +89,144 @@ const SignUpForm = () => {
   };
 
   useEffect(() => {
+    if (!checkEmail) return;
     setCheckEmail(false);
   }, [watchEmail]);
 
   useEffect(() => {
+    if (!checkNickname) return;
     setCheckNickname(false);
-  }, [watchNickName]);
+  }, [watchNickname]);
 
   return (
     <Container as='form' onSubmit={handleSubmit(onSubmit)}>
-      <FormControl isInvalid={!!errors.email}>
-        <FormLabel>Email</FormLabel>
-        <Flex>
-          <Input id='email' placeholder='Email' {...register('email')} />
-          <Button onClick={handleCheckEmail} colorScheme={checkEmail ? 'green' : 'red'}>
-            중복 확인
-          </Button>
-        </Flex>
-        <FormErrorMessage color='red'>{errors.email?.message}</FormErrorMessage>
-      </FormControl>
-
-      <FormControl isInvalid={!!errors.nickname}>
-        <FormLabel>Nickname</FormLabel>
-        <Flex>
-          <Input id='nickname' placeholder='Nickname' {...register('nickname')} />
+      <Stack spacing={4}>
+        <FormControl isInvalid={!!errors.email}>
+          <FormLabel fontSize='xs' fontWeight='bold' color={Common.colors.gray}>
+            Email
+          </FormLabel>
+          <InputGroup size='md'>
+            <Input id='email' placeholder='Email' {...register('email')} />
+            <InputRightElement>
+              {watchEmail && (
+                <MdCancel cursor='pointer' onClick={() => resetField('email')} />
+              )}
+            </InputRightElement>
+          </InputGroup>
+          <Flex>
```

Comment:
p4;
```suggestion
          <Flex justify='space-between'>
```
이렇게 바꾸면 중간에 `<Spacer/>` 태그를 지워도 되더라구요~